.github/workflows/nightly-pipeline.yml

```diff
@@ -66,18 +67,22 @@ jobs:
           image-path: aleph-release-synthetic-docker
           node-image: aleph-node:syntheticnet
           compose-file: docker/docker-compose.synthetic-network.yml
-        timeout-minutes: 60
+        timeout-minutes: 6
 
   check-e2e-test-suite-completion:
     needs: [
       run-e2e-high-out-latency,
       run-e2e-no-quorum-without-high-out-latency,
     ]
     name: Check e2e test suite completion
+    if: always()
```

Comment:
Non-blocking, feel free to resolve: this most likely means that the job will be triggered even if the workflow is cancelled. `if: success() || failure()` or `if: '!cancelled()'` may be more suitable.

Reply:
This is a good point, cancelling can occur in below scenarios:
* dev click `Cancel workflow in UI
* dev triggers nightly by hand and then while it's still in progress, pushes changes to branch and triggers again - workflow in concurrency is set to cancel in progress
In the above cases, we don't want to send slack notifications. 
I tested that with `!cancelled()`, we still have the same original behaviour, ie if upstream job failed, then e2e check suite completion is not skipped